src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1334,8 +1399,13 @@ ExpressionPtr Visitor::visitTypesafe(Parser::AdditiveExpressionContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-ExpressionPtr Visitor::visitTypesafe(
-    Parser::MultiplicativeExpressionContext* ctx) {
+void Visitor::visit(Parser::StrangeMultiplicativeSubexprOfAdditiveContext*) {
+  // StrangeMultiplicativeSubexprOfAdditiveContext must not be visited.
+  AD_FAIL();
```

Comment:
Do those case perform correctly in the meantime?
Maybe we can rename this now to something that doesn't have "Strange" in the name.
Maybe that's something to talk about in a meeting.

Reply:
Tbh I haven't tried to understand it completely. Its purpose is to detect if there is an operand (`+`,`-`) directly before/after a number without a space. But I currently don't know how exactly this works. From my experience it seems to be working correctly though.

Reply:
Okay, I have just read the standard and the grammar concerning this, and it is not too hard to fix.
For this PR you can leave it as is. I still have to think about the cleanest implementation of the `visit` functions. 
Maybe you can help me with some ANTLR hints in our next meeting.